VocaDbModel/Service/Queries/TagUsageQueries.cs

```diff
@@ -95,7 +96,10 @@ public class TagUsageQueries {
 					entryLinkFactory.CreateEntryLink(entry), string.Join(", ", tagNames)), user);
 
 				var addedTags = appliedTags.Except(entry.Tags.Tags).ToArray();
-				new FollowedTagNotifier().SendNotifications(ctx, entry, addedTags, new[] { user.Id }, entryLinkFactory, enumTranslations);
+
+				if (!((entry is SongList songList) && !songList.FeaturedList)) {

```

Comment:
Perhaps extract this to a private method bool AllowNotifications(entry). I might also consider moving the check to IEntryWithTags, but since it's just this one place, maybe it's ok to have the type check here.

Reply:
Thank you for your review. Since I think your solution is more elegant than mine, I added `AllowNotifications` property to `IEntryWithTags` interface. I wanted to use default interface implementation for the property, but the target runtime doesn't support it. I wrote a unit test for this, could you please review it?

Reply:
Default interface implementation would be great here, but unfortunately it's only for .NET Core 3.1, and we're not quite there yet (maybe some day).